docs/reference/metadata.md

```diff
@@ -175,37 +175,77 @@ Indentation options control aspects of the auto-indentation mechanism.
 `increaseIndentPattern`
 : *Regex.*
   If it matches on the current line,
-  the next line will be indented one level further.
+  lines after current line will be indented one level further.
+
+  If in `.tmPreferences`:
 
   ```xml
   <key>increaseIndentPattern</key>
-  <string>insert regex here</string>
+  <string><![CDATA[^\s*increase$]]></string>
+  ```
+
+  Then in `view`:
+
+  ```
+  line1
+  line2
+  increase
+    after increase1
+    after increase2
   ```
 
 `decreaseIndentPattern`
 : *Regex.*
   If it matches on the current line,
-  the next line will be unindented one level.
+  current line and lines after current line will be unindented one level.
+
+  If in `.tmPreferences`:
 
   ```xml
   <key>decreaseIndentPattern</key>
-  <string>insert regex here</string>
+  <string><![CDATA[^\s*decrease$]]></string>
+  ```
+
+  Then in `view`:
+
+  ```
+    line1
+    line2
+  decrease
+  after decrease1
+  after decrease2
   ```
 
 `bracketIndentNextLinePattern`
 : *Regex.*
   If it matches on the current line,
-  only the next line will be indented one level further.
+  only the next non-whitespace line will be indented one level further.
+
+  If in `.tmPreferences`:
 
   ```xml
   <key>bracketIndentNextLinePattern</key>
-  <string>insert regex here</string>
+  <string><![CDATA[^\s*bracket$]]></string>
+  ```
+
+  Then in `view`:
+
+  ```
+  line1
+  line2
+  line3
+  bracket
+    
+    first non-whitespace line after bracket
+  after bracket2
+  after bracket3
   ```
 
 `disableIndentNextLinePattern`
 : *Regex.*
   If it matches on the current line,
   the next line will not be indented further.
+  Deprecated in ST3, use unIndentedLinePattern instead.
```

Comment:
It's not deprecated as far as I understand, `unIndentedLinePattern` means the line doesn't affect the indentation calculation of other lines, but `disableIndentNextLinePattern` specifically disables just the `bracketIndentNextLinePattern` indentation. This is so that an `if` statement for example without a curly brace will indent the next line, but that indentation will be cancelled when typing `{`.